modules/Bio/EnsEMBL/Compara/Utils/MasterDatabase.pm

```diff
@@ -824,7 +824,17 @@ sub create_multiple_wga_mlsss {
     if ($method->type eq 'CACTUS_HAL') {
         my $pw_method = $compara_dba->get_MethodAdaptor->fetch_by_type('CACTUS_HAL_PW');
         push @mlsss, @{ create_mlsss_on_pairs($pw_method, $species_set->genome_dbs, $source, $url) };
+    } 
+    
+    if ( $no_release ) {
+        my @nr_mlsss;
+        foreach my $mlss ( @mlsss ) {
+            $mlss->{_no_release} = $no_release;
+            push @nr_mlsss, $mlss;
```

Comment:
What is this doing ? It feels like you're essentially copying `@mlsss` into `@nr_mlsss`. Both will have exactly the same content.

Reply:
It's just a hangover from previous experiences dealing with references. I don't trust in-place updates. Have changed the code.